conanfile.py

```diff
@@ -1,79 +1,91 @@
-from conans import ConanFile, tools
-import shutil
-import os.path
+# -*- coding: utf-8 -*-
+import os
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
 
 
 class RubyInstallerConan(ConanFile):
     name = "ruby_installer"
     version = "2.3.3"
-    api_version = "2.3.0"
     license = "MIT"
```

Comment:
is it MIT or GPL? package_info copies GPL file

Reply:
oh, that's a good question, I need to check. good catch

Reply:
Ruby license: https://spdx.org/licenses/Ruby.html

Reply:
done